data-node/src/main/java/org/graylog/datanode/initializers/JwtTokenValidator.java

```diff
@@ -43,28 +46,18 @@ public JwtTokenValidator(@Named("password_secret") final String signingKey) {
     public void verifyToken(String token) throws TokenVerificationException {
         SignatureAlgorithm signatureAlgorithm = SignatureAlgorithm.HS256;
```

Comment:
`SignatureAlgorithm` is deprecated. Let's use the recommended class.

Reply:
Fixed, thx!